keyring.go

```diff
@@ -70,7 +70,7 @@ func (b *KeyRing) DerivePrivKey(keyDesc keychain.KeyDescriptor) (
 // k is our private key, and P is the public key, we perform the following
 // operation:
 //
-//  sx := k*P s := sha256(sx.SerializeCompressed())
+// sx := k*P s := sha256(sx.SerializeCompressed())
```

Comment:
I had to do that because now `golangci/golangci-lint-action@v3` use the new version of golangci-lint (1.48.0).

This double space was detected as an error 

NOW:                `Requested golangci-lint 'latest', using 'v1.48.0'`
BEFORE:          `Requested golangci-lint 'latest', using 'v1.47.3'`

Reply:
Good catch, thank you!